### DISCUSSION TITLE
How to remove label from PR when the workflow triggered using the PR label is success or failed!!

### DISCUSSION THREAD

**Discussion by Himanshu-upadhyay7777** [community/community#46839] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

How to remove label from PR when the workflow triggered using the PR label is success or failed .I want to automatically remove the PR label after the workflow triggred by the PR label is success or failed

**Comment by jge162** [comment, marked as answer]
actions/remove-label See link [here](https://github.com/actions-ecosystem/action-remove-labels) @Himanshu-upadhyay7777 this should work for you.
